src/pages/settings/Subscription/SubscriptionPlan.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {Linking, Platform, View} from 'react-native';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useSubscriptionPlan from '@hooks/useSubscriptionPlan';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+function SubscriptionPlan() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    const subscriptionPlan = useSubscriptionPlan();
+
+    const isCollect = subscriptionPlan === CONST.POLICY.TYPE.TEAM;
+    // TODO: replace this with a real value once OpenSubscriptionPage API command is implemented
+    const isAnnual = true;
+
+    const benefitsList = isCollect
+        ? [
+              translate('subscription.yourPlan.collect.benefit1'),
+              translate('subscription.yourPlan.collect.benefit2'),
+              translate('subscription.yourPlan.collect.benefit3'),
+              translate('subscription.yourPlan.collect.benefit4'),
+              translate('subscription.yourPlan.collect.benefit5'),
+              translate('subscription.yourPlan.collect.benefit6'),
+          ]
+        : [
+              translate('subscription.yourPlan.control.benefit1'),
+              translate('subscription.yourPlan.control.benefit2'),
+              translate('subscription.yourPlan.control.benefit3'),
+              translate('subscription.yourPlan.control.benefit4'),
+              translate('subscription.yourPlan.control.benefit5'),
+              translate('subscription.yourPlan.control.benefit6'),
+              translate('subscription.yourPlan.control.benefit7'),
+          ];
+
+    let priceInfo;
+
+    if (isCollect) {
+        priceInfo = isAnnual ? translate('subscription.yourPlan.collect.priceAnnual') : translate('subscription.yourPlan.collect.pricePayPerUse');
```

Comment:
how about moving this condition inside translate method?

priceInfo = translate(`subscription.yourPlan.collect.${isAnnual ? 'priceAnnual' : 'pricePayPerUse'}`);

same thing below